docs/usage/merge-confidence.md

```diff
@@ -0,0 +1,97 @@
+# Merge Confidence
+
+Look at the Merge Confidence badges before merging to:
+
+- Prevent updates which break in production
+- See at a glance if you should update
+
+Merge Confidence finds and flags undeclared breaking releases.
+It analyzes test and release adoption data across Mend Renovate’s early-adopting user base.
+
+![Renovate PR with Merge Confidence badges](assets/images/merge-confidence.png)
+
+## Pull request badges
+
+Merge Confidence adds the following badges to your pull requests:
+
+- **Age**: The age of the package
+- **Adoption**: The percentage of this package's users (within Renovate) which are using this release
+- **Passing**: The percentage of updates which have passing tests for this package
+- **Confidence**: The confidence level for this update
+
+## Supported platforms
+
+Merge Confidence badges for pull requests are available on any supported platform or Renovate distribution, including Mend Remediate.
+
+## Supported languages
+
+Data is available for packages from:
+
+- npm
+- Maven
+- PyPI
+
+We plan to support more languages soon.
+
+## Enabling and disabling
+
+If you use the Mend Renovate App then the badges are enabled automatically.
+
+If you don't use the app, you can enable the badges by adding this to your Renovate config:
+
+```json lines
```

Comment:
@HonkingGoose does `mkdocs` supports `json lines` ?
couldnt find anything documented about this. 

if it does not. then we should either use `json` and surround this lines with curly brackets:

```json
{
    "extends": [
        "mergeConfidence:default"
     ]
}
```
or use `js`:
```js
"extends": [
    "mergeConfidence:default"
  ]
```